noxconfig.py

```diff
@@ -54,7 +54,7 @@ class Config:
     bucketfs_port = 2580
     connectors = ["pyodbc", "turbodbc", "websocket"]
     python_versions = ["3.9", "3.10", "3.11", "3.12", "3.13"]
-    exasol_versions = ["7.1.17"]
+    exasol_versions = ["7.1.30", "8.29.6"]
```

Comment:
We are not able to use more recent Exasol DB versions, as we are currently pinned to `exasol-integration-test-docker-environment==3.4.0`.  If we were to drop support for Python 3.9.x (EOL 2025-10-31), then we could update to `pytest-exasol-backend = "^1.0.0"` and `exasol-integration-test-docker-environment = "^4.0.0"`.

The last release of `sqlalchemy-exasol` version 5.1.0 was 2025-02-14. `sqlalchemy` 2.x has been out since 2023. 

While we should drop support for Python 3.9.x in the next few months, it seems like we should consider the potential customer impact if during the sqlalchemy migration if we suddenly dropped it. Maybe having some insight into our customers' usage would be helpful here.

Reply:
Decided to update to encourage good security practices with customers.